DOCS/man/options.rst

```diff
@@ -3937,6 +3937,13 @@ Demuxer
     libarchive opens all volumes anyway when playing the main file, even though
     mpv iterated no archive entries yet.
 
+``--directory-mode=<recursive|lazy|ignore>``
+    When opening a directory, open subdirectories recursively, lazily or not at
+    all (default: recursive).
+
+    Disabling this can lead to problems with resuming playlists
+    (`RESUMING PLAYBACK`_) and possibly other things.
```

Comment:
Sorry I guess this should probably be reworded too. How about, `Values other than ``recursive`` can lead to problems...`